Api/ride.py

```diff
@@ -49,35 +45,30 @@ def get(self,location):
         for offer in ride_Offers:
             if offer["location"] == location:
                 list_of_offers.append(offer)
-            return {"list of offers":list_of_offers},200
+        return {"list of offers":list_of_offers},200
     
 class DriverRideOffer(Resource):  
     def post(self):
```

Comment:
A good comment within a function should tell me what parameters the functions take and the expected return value. For example, `"Takes in `username` and `password` and return a ` boolean` value"`